### ISSUE TITLE
Some thing I do not understand

### ISSUE THREAD

**Issue by Methen** [Feodor2/Mypal68#464]
This is something that just makes no sense to me, I keep the Mypal browser in my documents folder and a shortcut on the desktop, Now here is where it gets strange I down  load the latest version of mypal and unzip it on the desk top I go into the folder and click on the short cut and open the browser I enable the book mart tool bar and guess what the book marks are already  in there and that to me simply does not make any sense I did not install them this is a new version and its on the desk top so how did they get in there ?

**Comment by bluedragon-cairo** [comment]
Maybe you are not using the portable version?

**Comment by RamonUnch** [comment]
when you extract anywhere mypal it will look by default in your user profile for his profile, so if you want several independent install of MyPal, you must specify the profile on the command line (`-p` command) or use the portable mode:
https://github.com/Feodor2/Mypal68/wiki/Portable

**Comment by Methen** [comment]
I am using the desktop version and default user profile is in my  doculments folder not on the desktop and I not using the portable version

**Comment by zanud** [comment]
> and default user profile is in my doculments folder not on the desktop

In non-portable mode Mypal by default places user profile into the `<Windows_user_profile_folder>\Application Data\Mypal68\Profiles` folder (on WinXP, or corresponding folder on Win7+).

Any other place requires either a command line parameter or portable mode.

**Comment by Methen** [comment]
Yes but that would only make sense if you actually installed it what I am using is a zip file that I simply unzipped on the desk top and click on the short cut  I did not install it like a regular executable

**Comment by RamonUnch** [comment]
> Yes but that would only make sense if you actually installed it what I am using is a zip file that I simply unzipped on the desk top and click on the short cut I did not install it like a regular executable


Any exe can look for %APPDATA% you do not need installation for this and most installers simply extract the file to a directory and add an add/remove option in the program list plus start menu entry and a desktop shortcut. zipped version does not mean portable version.

Type `about:profiles` in the URL bar of mypal and you will see the profile folders for yourself.

**Comment by zanud** [comment]
@Methen:

You emphasized a few times that existed Mypal profile was in the My Documents folder, whereas it's impossible without a special command line parameter.

**Comment by JimMP68User** [comment]
> Type `about:profiles` in the URL bar of mypal and you will see the profile folders for yourself.

The little known secret - that few know about. +1

**Comment by zanud** [comment]
There is yet one "little known secret": default profile directory name depends on Mypal's directory name.

Try the following:
1. Run Mypal and look on the `about:profiles` page the directory name of a currently used profile.
2. Exit Mypal.
3. Rename Mypal's directory.
4. Run Mypal and look on the `about:profiles` page the directory name of a currently used profile.
5. Exit Mypal and rename Mypal's directory back.

So Mypal unpacked to the Desktop and Mypal unpacked to the "Program files" will use different profiles.
Sometimes this is useful, sometimes weird.

**Comment by Feodor2** [comment]
So hope finally it came to understand.
Anyway this is not about Mypal only.
